hdr_html_canvas_element.md

```diff
@@ -39,40 +39,44 @@ Add a new attribute to `ScreenAdvanced` to indicate the HDR headroom currently a
   }
 ```
 
-The high dynamic range headroom of a display is computed as:
+See the below discussion of the display of floating point color values for more details.
```

Comment:
Perhaps: "See below for more details of the display of floating point color values"?

Reply:
Good idea -- updated.